engines/sci/engine/workarounds.cpp

```diff
@@ -412,6 +412,7 @@ const SciWorkaroundEntry uninitializedReadWorkarounds[] = {
 	{ GID_SQ1,            -1,   703,  0,         "firePulsar", "changeState",     sig_uninitread_sq1_1,     0, { WORKAROUND_FAKE,   0 } }, // export 1, but called locally (when shooting at aliens)
 	{ GID_SQ4,            -1,   398,  0,            "showBox", "changeState",                     NULL,     0, { WORKAROUND_FAKE,   0 } }, // CD: called when rummaging in Software Excess bargain bin
 	{ GID_SQ4,            -1,   928, -1,           "Narrator", "startText",                       NULL,  1000, { WORKAROUND_FAKE,   1 } }, // CD: happens in the options dialog and in-game when speech and subtitles are used simultaneously
+	{ GID_SQ4,           395,   395,  -1,   "fromStoreScript", "changeState",                     NULL,    -1, { WORKAROUND_FAKE,   0 } }, // CD: happens, when shoplifting in Galaxy Galleria  - bug #10229
```

Comment:
There is a whitespace error in the comment before the hyphen, and the index should be 0, not -1, unless this function is using many uninitialised temps.

Reply:
vv t shows at least 3 uninitialised temps at that point, but I don't know if it uses the others or not. Should I still change the index to 0?

Reply:
Temps are always uninitialised at the beginning of a function. This is normal. If an uninitialised read occurs, the engine will raise an error just like the one from the original ticket, which then needs to get evaluated and the correct fake value entered into the workarounds table.